src/popup/popup.html

```diff
@@ -104,7 +104,7 @@
         <h3 class="text-align-center vocab heading">Bookmarks</h3>
         <button class="delete-all-button vocab button negative-indicating dark-shaded normal-sized basic">
```

Comment:
Since we have changed the context of the "delete" button, change the class name also from `delete-all-button` -> `delete-bookmarks-button`.

Check if any stylesheet rules or javascript code depends on that particular class and modify it accordingly.

Reply:
Hey, @makkoncept  I have changed the files as you directed.